src/contexts/messageInputContext/MessageInputContext.tsx

```diff
@@ -429,6 +440,13 @@ export const MessageInputProvider = <
   value: InputMessageInputContextValue<At, Ch, Co, Ev, Me, Re, Us>;
 }>) => {
   const { client } = useChatContext<At, Ch, Co, Ev, Me, Re, Us>();
+  const {
+    closePicker,
+    openPicker,
+    selectedPicker,
+    setSelectedPicker,
+  } = useAttachmentPickerContext();
```

Comment:
just a small reordering 😬 
```suggestion
  const {
    closePicker,
    openPicker,
    selectedPicker,
    setSelectedPicker,
  } = useAttachmentPickerContext();

    const { client } = useChatContext<At, Ch, Co, Ev, Me, Re, Us>();
```